arangodb-net-standard/CollectionApi/CollectionApiClient.cs

```diff
@@ -49,14 +47,23 @@ public async Task<DeleteCollectionResponse> DeleteCollectionAsync(string collect
             throw new ApiErrorException(error);
         }
 
-        public async Task PutCollectionTruncateAsync(string collectionName)
+        /// <summary>
+        /// Truncates a collection, i.e. removes all documents in the collection.
+        /// PUT/_api/collection/{collection-name}/truncate
```

Comment:
What about putting this in the remark section? Currently it will show up in the intellisense which I find a bit annoying, but maybe that was the intent here.

Reply:
There's a general approach in this project of not abstracting away details of the REST API. But I did change the method name here because... well, `PutCollectionTruncateAsync` is a plain ugly name IMO.  But this does possibly obscure which endpoint this relates to. So putting the api endpoint in the summary was a conscious choice, allowing a user to more easily find out which particular endpoint this method maps to.  It surprises me to hear you might find it annoying though. Why do you think it is annoying?

Reply:
That's a good point. Thinking about it again, it makes sense actually. I remember how we removed these in the docs of web API endpoints because they end up in the specs and intellisense, but here we are on the other side 😄